src/graphql/documents/projectEvents.graphql

```diff
@@ -21,6 +21,11 @@ query ProjectEvents(
       amount
       note
       terminal
+      projectId
```

Comment:
adding these properties to eventually be passed to `ActivityElement` component to optionally render a project link